.circleci/config.yml

```diff
@@ -50,43 +50,47 @@ workflows:
     jobs:
       - build:
           <<: *common_filters
+      - skip_check:
```

Comment:
I'm okay with the `[skipTests]` name in the commit message, but I find the CI job name a bit unintuitive when looking at the screenshot since the name is `skip_check` but it fails if you want to skip the check.

Proposing something like `runFullBuild`? So it fails if there's `[skipTests]` in the commit message and passes if there's no `skipTests`